samples/snippets/create_mysql_connection.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START bigqueryconnection_connection_create]
+from google.cloud import bigquery_connection_v1 as bq_connection
+
+"""This sample shows how to create a BigQuery connection with a Cloud SQL for MySQL database"""
+
+
+def main() -> None:
+    # TODO(developer): Set project_id to the project ID containing the
+    # connection.
+    project_id = "your-project-id"
+    # TODO(developer): Set location to the location of the connection.
+    # See: https://cloud.google.com/bigquery/docs/locations for a list of
+    # available locations.
+    location = "US"
+    # TODO(developer): Set database to the name of the database in which you're creating the connection.
+    database = "my-database"
+    # TODO(developer): Set instance to the instance where you're creating the connection.
+    instance_name = "my-instance"
+    # TODO(developer): Set instance_location to the location of the instance where you are creating the connection.
+    instance_location = "my-instance-location"
+    # TODO(developer): Set username to the database username.
+    username = "my-username"
+    # TODO(developer): Set password to the database password.
+    password = "my-password"
+    cloud_sql_conn_name = f"{project_id}:{instance_location}:{instance_name}"
```

Comment:
I don't think you need to have the user input these as 3 different fields. You can just ask them for the "instance connection name". This is a specific field in the instance metadata and is shown to the user in console, e.g.:
![image](https://user-images.githubusercontent.com/31518063/162079704-8d80c528-a9df-43ea-8ebb-450f9f92241c.png)


For reference, this is what we do in Cloud SQL samples:

https://github.com/GoogleCloudPlatform/python-docs-samples/blob/HEAD/cloud-sql/mysql/sqlalchemy/main.py#L160-L173